src_py/__init__.py

```diff
@@ -129,6 +130,7 @@ def warn(self):
 
 try:
     import pygame.event
+    from pygame.event import Event
```

Comment:
If the event module is not available, the Event class won't exist. Other modules where the import fails, they define a dummy function (see Cursor for example).